Started adding to en.html

snengle opened the pull request.

mlosde commented:
Are you sure about the translations like "Orc Captain" or "Gloomy Forest"? Isn't it "orc King" and "Dark Forest"?
ToC is great tho

snengle commented:
You are correct. I haven't fixed all the issues with the TOC yet. I will continue to fix it as I work on the related sections.

mlosde commented:
Perfect, I'd propose that we simply let this PR open, so you can make your commits and we can discuss the changes here.

snengle commented:
Sounds good. I just found that the combat skill Mirror Image is missing from de.html (I'm new to the game so I assume that is a recent addition)

snengle commented:
I have completed the English translation. I don't know German so I was mainly using Google Translate and then fixing things from there. I have only been playing the game for a little over a month so I'm sure I missed somethings but I think it is pretty good.

I did make a couple changes that needed to be updated:

- Changed attacks per battle from 4 to 3
- Changed coins per orc kill from 2 to 10

There are some other things I'm unsure about or are missing from the German version that I didn't change:

- Missing combat skill Mirror Image
- Unsure about what level Coin Carrier shows up (it was like 150 for me)
- Unsure about what level Serf Trader shows up (I still haven't seen him)
- Missing Orc Mud Crawler
- Does Increased Bounty still exist
- Missing Ring of Elemental Forces in blue prints
- Missing items Root Trap, Thorium Breastplate and Thorium Greaves

ChrisDuffley commented:
Thank you for your contribution!

@mlosde / @snengle: If you feel better just keeping this PR open, do you want me as a collaborator in your repository, this way I can make changes while this is open? I can help you answer at least some of your questions such as level of orc surftraders etc.

mlosde commented:
Thanks, hopefully I have the time for further review on the weekend. @ChrisDuffley You have to ask the developers of knight manager if you can collaborate directly on the repo, as all things that are pushed to main will automatically go live on the website which is maintained by Krögoor.
